package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "leaderboard",
+  "version": "1.0.0",
+  "description": "The leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external [Leaderboard API service.",
+  "private": true,
+  "scripts": {
+    "build": "webpack",
+    "start": "webpack serve --open"
```

Comment:
- [ ] The `webpack` script is not working due to the wrong config in the webpack, please fix it.

![jj](https://user-images.githubusercontent.com/61562626/136007479-4ba74a26-d7a1-40e7-a9fb-7771c325f230.png)

Reply:
Thanks for the review. I have made the required changes